daemon/mgr/container_stats.go

```diff
@@ -81,6 +81,11 @@ func (mgr *ContainerManager) Stats(ctx context.Context, name string) (*container
 	ID := c.ID
 	c.Unlock()
 
+	// only get metrics when the container is running
+	if !(c.IsRunning() || c.IsPaused()) {
+		return nil, nil, nil
```

Comment:
`return nil` will panic if the StreamStat `toContainerStats` consumes the result.

Reply:
make sense, i have fix it.